DESCRIPTION.md

```diff
@@ -500,6 +500,30 @@ If you get an error telling you "The request signature is invalid", please check
 - You use the data to sign to create a SHA256 hash signature.
 - You have base64 encoded the SHA256 hash signature before adding it to the request under `X-Bunq-Client-Signature`.
 
+**Note:** Though request signing is a must on production, you can choose to disable it on sandbox to simplify the testing. Here's how it works:
+
+1. Set the `X-Bunq-Client-Signature-Validation-Policy` header of the request to `IGNORE_ONLY_FOR_TESTING`.
```

Comment:
This is now `X-Bunq-Client-Signature` :)

Reply:
True. Fixed it!